### DISCUSSION TITLE
Reason source formatting not working in IntelliJ IDEA 2024.1.1

### DISCUSSION THREAD

**Discussion by faseidl** [giraud/reasonml-idea-plugin#466] [General]
I have recently installed the ReasonML plugin 0.124-2024.1 but I am not able to reformat ReasonML code. I notice that Reason does not appear in the Code Styles setting in IntelliJ. Shouldn't it be?
<img width="227" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/f195bb6a-421b-48eb-bc04-bdb5c2aa6fab">

**Comment by giraud** [comment]
Yes it should. 
Is the plugin correctly listed in the plugins settings ?

**Reply by faseidl** [reply]
Yes, it is listed:
<img width="989" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/4503f3ef-39b9-4ded-96e1-0c230e896b5f">

**Reply by giraud** [reply]
You should probably reinstall it, or update to 0.125 for that.
Also look in idea logs if there are any errors regarding the plugin.

**Reply by faseidl** [reply]
Thx. I tried re-installing, but I only see 0.124 available (via the Marketplace in Intellij). I s0.125 available somewhere else?

**Reply by giraud** [reply]
My bad, there is no 0.125 yet
Do you have exceptions in idea logs ?

**Reply by faseidl** [reply]
Yes, I do have exceptions:
<img width="1581" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/076e9b34-6b3b-4209-9d9c-b46dc510a805">
<img width="1581" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/dbb904a9-2fcf-4413-9bae-8d96ab8d90ee">
<img width="1581" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/7253a85e-7547-439c-a6d3-ff0655e89f47">
<img width="1581" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/e4a140fb-1564-484c-9810-3b91724856fa">
<img width="1581" alt="image" src="https://github.com/giraud/reasonml-idea-plugin/assets/174294/e27cabc7-d251-4a78-8def-b73f05f6b498">

**Reply by giraud** [reply]
The 'blockless function' exception is harmless but I tried to fix it in #465, apparently it’s not 🙂 .

I never had the 'comparison' exception. Do you have a reproducible code that generate it ?
